src/main/java/org/opensearch/knn/index/query/ExactSearcher.java

```diff
@@ -233,7 +238,7 @@ public static class ExactSearcherContext {
          */
         boolean useQuantizedVectorsForSearch;
         int k;
-        BitSet matchedDocs;
+        DocIdSetIterator matchedDocs;
```

Comment:
probably rename this to `matchedDocsIterator`.

Reply:
Ack